client/src/Components/Nav.jsx

```diff
@@ -0,0 +1,34 @@
+import 'font-awesome/css/font-awesome.min.css';
```

Comment:
I'm not sure it's a good idea to import all of font-awesome here. You will then be downloading the entire font-awesome library. React-icons addresses this problem.